tests/unit/test_sensors.py

```diff
@@ -0,0 +1,400 @@
+    # expected measurement without noise
+    ax, ay, az = total_rotation @ (rocket_rotation @ accel)
+    # expected measurement with constant bias
+    ax += 0.5
+    ay += 0.5
+    az += 0.5
+
+    # check last measurement considering noise error bounds
+    noisy_rotated_accelerometer.measure(t, u, UDOT, relative_position, gravity)
+    assert noisy_rotated_accelerometer.measurement == approx([ax, ay, az], rel=0.5)
+
+
+def test_noisy_rotated_gyroscope(noisy_rotated_gyroscope):
+    """Test the measure method of the Gyroscope class. Checks if saved
+    measurement is (wx,wy,wz) and if measured_data is [(t, (wx,wy,wz)), ...]
+    """
+    t = SOLUTION[0]
+    u = SOLUTION[1:]
+    # calculate acceleration at sensor position in inertial frame
+    relative_position = Vector([0.4, 0.4, 1])
+    gravity = 9.81
+    omega = Vector(u[10:13])
+    # calculate total rotation matrix
+    cross_axis_sensitivity = Matrix(
+        [
+            [1, 0.005, 0.005],
+            [0.005, 1, 0.005],
+            [0.005, 0.005, 1],
+        ]
+    )
+    sensor_rotation = Matrix.transformation(euler_to_quaternions(-60, -60, -60))
+    total_rotation = sensor_rotation @ cross_axis_sensitivity
+    rocket_rotation = Matrix.transformation(u[6:10])
+    # expected measurement without noise
+    wx, wy, wz = total_rotation @ (rocket_rotation @ omega)
+    # expected measurement with constant bias
+    wx += 0.5
+    wy += 0.5
+    wz += 0.5
+
+    # check last measurement considering noise error bounds
+    noisy_rotated_gyroscope.measure(t, u, UDOT, relative_position, gravity)
+    assert noisy_rotated_gyroscope.measurement == approx([wx, wy, wz], rel=0.5)
+
+
+def test_quatization_accelerometer(quantized_accelerometer):
+    """Test the measure method of the Accelerometer class. Checks if saved
+    measurement is (ax,ay,az) and if measured_data is [(t, (ax,ay,az)), ...]
+    """
+    t = SOLUTION[0]
+    u = SOLUTION[1:]
+    # calculate acceleration at sensor position in inertial frame
+    relative_position = Vector([0, 0, 0])
+    gravity = 9.81
+    a_I = Vector(UDOT[3:6])
+    omega = Vector(u[10:13])
+    omega_dot = Vector(UDOT[10:13])
+    accel = (
+        a_I
+        + Vector.cross(omega_dot, relative_position)
+        + Vector.cross(omega, Vector.cross(omega, relative_position))
+    )
+
+    # calculate total rotation matrix
+    rocket_rotation = Matrix.transformation(u[6:10])
+    # expected measurement without noise
+    ax, ay, az = rocket_rotation @ accel
+    # expected measurement with quantization
+    az = 2  # saturated
+    ax = round(ax / 0.4882) * 0.4882
+    ay = round(ay / 0.4882) * 0.4882
+    az = round(az / 0.4882) * 0.4882
+
+    # check last measurement considering noise error bounds
+    quantized_accelerometer.measure(t, u, UDOT, relative_position, gravity)
+    assert quantized_accelerometer.measurement == approx([ax, ay, az], abs=1e-10)
+
+
+def test_quatization_gyroscope(quantized_gyroscope):
```

Comment:
Here it should be "quantization". There are other places the misspelling occurs.